client/rpc/block_converter.go

```diff
@@ -0,0 +1,147 @@
+package rpc
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+	"github.com/tendermint/tendermint/version"
+	"sync"
+	"time"
+)
+
+type CosmosResultBlock struct {
+	BlockMeta CosmosBlockMeta `json:"block_meta"`
+	Block     CosmosBlock     `json:"block"`
+}
+
+type CosmosBlockMeta struct {
+	BlockID types.BlockID `json:"block_id"` // the block hash and partsethash
+	Header  CosmosHeader  `json:"header"`   // The block's Header
+}
+
+type CosmosHeader struct {
```

Comment:
if Tendermint header updates I believe it will break this?

Reply:
Probably, but IMO it's the only way we can have Bech32 validator addresses into the block header. 

Do you have any suggestion on how we can achieve that otherwise without replacing the header type definition?

Reply:
I think I got it and it's being discussed inside https://github.com/cosmos/cosmos-sdk/pull/4922#discussion_r315697469. 

Can we close this?